packages/flutter_tools/test/integration.shard/flutter_plugins_dependencies_test.dart

```diff
@@ -47,6 +46,7 @@ void main() {
         pluginAPath,
         '--template=plugin',
         '--project-name=plugin_a_real_dependency',
+        '--platforms=ios',
```

Comment:
Previously the plugin didn't support any platforms, which caused the `.flutter-plugins-dependencies` file to have no plugins per platform, which caused the test below to check nothing.